paasta_tools/check_mesos_resource_utilization.py

```diff
@@ -83,6 +85,11 @@ def check_thresholds(percent):
         over_threshold = True
     else:
         output += "OK: CPU usage is under %d%%. (Currently at %f%%)\n" % cpu_print_tuple
+    if current_disk >= percent:
+        output += "CRITICAL: Disk usage is over %d%%! Currently at %f%%!\n" % disk_print_tuple
```

Comment:
can you add the equivalent of https://github.com/Yelp/paasta/pull/237here please?